.changeset/mean-snakes-appear.md

```diff
@@ -0,0 +1,6 @@
+---
+'@clerk/localizations': minor
+'@clerk/themes': minor
+---
+
+Drop react as peer dependency from `@clerk/localizations` and `@clerk/themes`.
```

Comment:
```suggestion
Drop `react` and `react-dom` as peer dependencies since they are not necessary for this package
```

Changesets will already put the context of where they are removed from into the changelog 👍